src/DotVVM.Framework/Resources/Scripts/DotVVM.ts

```diff
@@ -1321,6 +1321,22 @@ class DotVVM {
             }
         };
 
+        ko.bindingHandlers["dotvvm-checkedItems"] = {
+            after: ko.bindingHandlers.checked.after,
+            init: ko.bindingHandlers.checked.init!.bind(this),
+            options: ko.bindingHandlers.checked.options,
```

Comment:
Ahhh :fearful: `this` is now different for `init`, `checked` and `update`

Reply:
I removed the calls to `bind` altogether since, after inspection of `applyBindingsToNodeInternal` in knockout a arrived at the conclusion that `this` is always the Window object both in `init` and `update`.